mmengine/visualization/vis_backend.py

```diff
@@ -354,17 +354,25 @@ class WandbVisBackend(BaseVisBackend):
                 updates the current metrics dict with the row argument
                 and metrics won't be saved until `wandb.log` is called
                 with `commit=True`. Default to True.
+        log_code_name: (str, optional) The name of code artifact.
```

Comment:
Hi, can we rename `log_code_name` to `log_code_kwargs` which is a dict passed to `log_code` method?

Reply:
We can, but `log_code` method only have four param, root, name and two callable. We can only get the root and name from func kwargs. That two callables are not easy to get from vis_backend init and from config file, I think. The root is set to current path by default, any reason to change it? At least, I can not figure it out.

The only reason for this change is that we can accept two lists of regular expressions to implement the include_func and exclude_func. But, it will take many lines code to implement this, which will be a little ugrly. On the other hand, we use the python file as config file instead of yaml file, so wandb will log all config files, it seems we have no reason to change the default include and exclude func. And, that's why I only put the name param here.

What's your opinion? If you persist in the dict format, I can give it a try.

Reply:
The `Registry` supports registering functions so it is possible to accept two strings to provide `include_func` and `exclude_func`. Although there is no difficulty in implementation, I rather agree with your idea that there is no need to support full features of `log_code` currently.

Reply:
Ok, I got it, but can you explain more about it? Is there any docs or code examples? I think I need some help to implement this.